### ISSUE TITLE
ecosystem/next-steps

### ISSUE THREAD

**Issue by jarrodmillman** [scientific-python/scientific-python.org#104]
https://github.com/scientific-python/scientific-python.org/blob/main/content/en/ecosystem/next-steps.md

This should provide pointers to the next steps a new user should take after installing.  It should mainly point to a few good resources and point out a few tools people will need to get started.  This text is from the old scipy.org site and should be rewritten.  It may be reasonable to discard all the existing text and reorganize things.

 - [ ] Consider moving the `Hello Scipy` section to the installation page.
 - [ ] ...
 - [ ] Record video

**Comment by stefanv** [comment]
> "Scientific Python" doesn't exist without "Python".

This may be confusing to newcomers.  I would be explicit and explain that Scientific Python is a set of tools, written in the Python language.

>  remember to have fun, make mistakes, and persevere.

Perhaps: Have fun, allow yourself to make mistakes, and—importantly—keep going.

I.e., not "remember to make mistakes".

The "Where to Write" section: I would flip things around and explain that Python is an interpreted language, i.e. that it reads files containing lines of text and executes them line-by-line.  You can make those files any way you like: most of us do so using a text editor (like Spyder or VS Code).  There's another way you can play around with Python: in a so-called Jupyter notebook—an interactive web application that lets you play around with Python snippets.

For "Hello SciPy": it may be difficult to show code examples in a video, especially this early on.  If we do this, I would start them with a real code snippet that does a real computation and plots the result.  Also be specific about filenames: e.g., say "save this as `test.py`" instead of `*.py`.

An example snippet could be something like:

```python
import numpy as np  # Import the NumPy library
import matplotlib.pyplot as plt

x = np.linspace(0, 1, 
y = np.sin(x**3)

plt.plot(x, y)
plt.show()
```

But an easier way to get them to play with code snippets might be to point them to the SciPy lecutures or to a gallery example from an existing project.

Pointing them to tutorials is good: probably in the style of "please see the links under the video".

In terms of learning by coding, I would emphasize the Python.org tutorial, the SciPy lectures, and perhaps a learning platform like Exercism.io (I haven't tried their Python course specifically, but I love their other courses—maybe we should add a scientific Python course there!).

The "stuck get help part" we may have to drop here—we cannot get into enough detail to truly help them.  I suggest we make another tech video about how to debug your problems.

How to find functions, however, is useful, but again probably quite a bit more sophisticated than covered in the rest of this video.  Perhaps a topic for the guide instead?

---

A thought: should this section perhaps rather be a little screencast-style demo of how to run your first Python program?  First showing how to use an editor to write a script, then run it in python, then in IPython, and finally in Jupyter.

Then, navigate to the SciPy lectures (or show it as backdrop), and also perhaps a few others sites (galleries maybe).

I think we should keep it fairly high level, and not get too much into the weeds.  For that, we have our guides and the tutorials themselves.

**Comment by jarrodmillman** [comment]
Closed by #145.
